addons/website_sale/views/snippets/snippets.xml

```diff
@@ -218,15 +218,22 @@
                          data-customize-website-views="website_sale.product_quantity"
                          data-no-preview="true"
                          data-reload="/"/>
+            <we-checkbox string="Tax Indication"
+                         data-customize-website-views="website_sale.tax_indication_opt"
```

Comment:
Actually it's my bad but it makes no sense to have two templates for this.
We can still check if it's active before calling it, we don't need the extra data for this.
Will fix it myself